index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
+    <title>Adopt A Dog</title>
+</head>
+<body>
+  <div class="grid-container">
+    <section id="banner">
+        <div class="filler">
+          </div>
+      <div id="letterhead">
+        <img src="images/adoptadog-logo.svg" id="toplogo" alt= "profile of a dog in white">
+        Seattle's premiere dog adoption agency
+      </div>
+      <div id="cart">
+        <div class="top-cart">
+          <img src="images/cart-icon.svg" id="carticon" alt="a white cart logo">
+          &nbsp;&nbsp;$1234.56
+        </div>
+        <div class="checkout-button">
+          Checkout
+        </div>
+      </div>
+      <div class="filler">
+        </div>
+    </section>
+    <section id="subbanner">
+      <div >
+        <p><a class="subtitle" href="index.html">Home&nbsp;&nbsp;/</a><a class="subtitle" href="dogs.html">&nbsp;&nbsp;Dogs&nbsp;&nbsp;/</a><a class="subtitle" href="blog.html">&nbsp;&nbsp;Blog</a></p>
+      </div>
+    </section>
+    <section id="welcome">
+      <div>
+        <img src="images/home-intro.jpg" id="hugpicture" alt="a dog appears to smile as it receives a hug">
+      </div>
+      <div class="lead-in">
+        <h2>Adopt A Dog</h2>
+        <p >
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed vitae lectus sit amet lorem semper vehicula. Etiam nunc neque, sollicitudin et est at, auctor accumsan urna. Suspendisse at nisl eget justo suscipit maximus. Aenean auctor urna nec est efficitur tristique. Cras pellentesque purus vel congue aliquam. Duis fringilla, lorem id vehicula sollicitudin, risus quam placerat ligula, aliquam laoreet risus augue sed mi. Aenean tincidunt, risus eget egestas efficitur, justo quam posuere justo, non elementum sem odio et felis. Praesent fringilla faucibus pellentesque. Nullam sit amet sapien fringilla
+        </p>
+      </div>
+    </section>
+    <section class="profiles">
+      <div class="profile-card">
+        <div>
+          <img src="images/murphy-card.jpg" class="profilepicture" alt="one of our many wonderful dogs">
+        </div>
+        <div>
+          <h3>Murphy</h3>
+          <p class="cost"><b>Cost to Adopt:</b>&nbsp;$123.45</p>
+          <p class="bio">Fusce vel elementum dolor. Nam dictum, purus nec commodo sodales, purus ligula faucibus felis, rhoncus interdum tellus libero ut leo. Ut pharetra magna</p>
+        </div>
+        <div class="adopt-button">
+          <a class="checkout" href="checkout.html">Adopt</a>
+        </div>
+      </div>
+      <div class="profile-card">
+          <div>
```

Comment:
the the profile cards are pretty div heavy, see if you can work without so many of them